wkcuber/tests/test_main.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+import numpy as np
+import pytest
+from wkcuber.utils import (
+    setup_logging,
+)
+from webknossos import Dataset
+from wkcuber.__main__ import create_parser, cube_with_args
+from tifffile import TiffWriter
+
+TESTOUTPUT_DIR = Path("testoutput")
+
+
+@pytest.mark.parametrize("category", ["color", "segmentation"])
+def test_main(category: str) -> None:
```

Comment:
Nice. I'm starting to think that we should port your bash test scripts also to pytest. Asserting with the Dataset API makes a lot of sense.

Reply:
Yes, I agree. I'll highlight this point in https://github.com/scalableminds/webknossos-libs/issues/580, too.